impeller/typographer/text_frame.cc

```diff
@@ -8,30 +8,13 @@ namespace impeller {
 
 TextFrame::TextFrame() = default;
```

Comment:
The default constructor is no glyphs, empty bounds, no color. Which should be perfectly safe.